### PR TITLE
Add caption support to amp-instagram

### PR DESCRIPTION
**Pull request by jpettitt**
Add support for captions to `amp-instagram` via the `data-captioned` attribute

Fixes captions in issue #8242

### CODE REVIEW

**Comment by jpettitt** [comment]
CLA signed under Relay Media Inc

File: extensions/amp-instagram/0.1/amp-instagram.js [hunk 1]
```diff
@@ -92,6 +95,8 @@ class AmpInstagram extends AMP.BaseElement {
         this.element.getAttribute('shortcode')),
         'The data-shortcode attribute is required for <amp-instagram> %s',
         this.element);
+    this.captioned_ = this.element.hasAttribute('data-captioned') ?
```
Comment (cramforce) [inline extensions/amp-instagram/0.1/amp-instagram.js]: FMI: This will essentially always trigger resize, right?

Comment (jpettitt) [inline extensions/amp-instagram/0.1/amp-instagram.js]: Yes.

**Comment by jpettitt** [comment]
@cramforce How do I make the bot understand that I signed the CLA as Relay Media? Or do I just need to sign it again personally?

**Comment by cramforce** [comment]
@jpettitt Do you have your GitHub account associated with a RelayMedia email address? If not, that should do it.

File: extensions/amp-instagram/0.1/amp-instagram.js (hunk 1 above)
Comment (cramforce) [inline extensions/amp-instagram/0.1/amp-instagram.js]: @jpettitt unless an awesome CMS knows the right dimensions :)

**Comment by jpettitt** [comment]
@cramforce I added my relaymedia email to github so hopefully it will verify now.

**Comment by cramforce** [comment]
@jpettitt Please push a new commit (hash) to retrigger the bot.

**Comment by jpettitt** [comment]
@cramforce done.  Ready for code review assuming Travis doesn't barf the comment change I just made.

**Comment by cramforce** [comment]
Thank you!!!

**Pull request merged**